applications/FluidDynamicsApplication/custom_processes/shock_capturing_entropy_viscosity_process.h

```diff
@@ -298,7 +299,8 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingEntropyViscosityProce
     static std::tuple<TotalDerivativeUtil, TotalDerivativeUtil, Vector> BuildTotalDerivativeUtils(
         const Element& rElement,
         const double DeltaTime,
-        const double HeatCapacityRatio);
+        const double HeatCapacityRatio,
+        const double c_v);
```

Comment:
```suggestion
        const double SpecificHeatCV);
```
Be consistent among similar functions, this eases going through the code later on.